letshelp-letsencrypt/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+from setuptools import find_packages
+
+
+install_requires = ["mock<1.1.0",]
```

Comment:
Please add a comment that this constraint is due to Python2.6.

Actually I would prefer to have this throughout the entire code base:
```python
install_requires = []
if sys.version_info < (2, 7):
    install_requires.append("mock<1.1.0")
else:
    install_requires.append("mock")
```
but if you think it would be incoherent without updating other `setup.py` scripts, feel free to just leave a comment